src/test/java/org/mifos/integrationtest/cucumber/stepdef/GSMATransferStepDef.java

```diff
@@ -97,6 +97,30 @@ public void callCreateSavingsAccountEndpoint() throws JsonProcessingException {
         logger.info("Savings Account Response: " + gsmaTransferDef.responseSavingsAccount);
         assertThat(gsmaTransferDef.responseSavingsAccount).isNotEmpty();
     }
+    @Then("I call the interop identifier endpoint")
+    public void callCreateInteropIdentifierEndpoint() throws JsonProcessingException {
+        // Setting headers and body
+        RequestSpecification requestSpec = Utils.getDefaultSpec();
+        requestSpec = gsmaTransferDef.setHeaders(requestSpec);
+        gsmaTransferDef.interopIdentifierBody = gsmaTransferDef.setBodyInteropIdentifier();
+        // Setting account ID in path
+        SavingsAccountResponse savingsAccountResponse = objectMapper.readValue(
+                gsmaTransferDef.responseSavingsAccount, SavingsAccountResponse.class);
+        gsmaConfig.interopIdentifierEndpoint = gsmaConfig.interopIdentifierEndpoint.replaceAll("\\{\\{payer_identifierType\\}\\}", "ACCOUNT_ID");
+        gsmaConfig.interopIdentifierEndpoint = gsmaConfig.interopIdentifierEndpoint.replaceAll("\\{\\{payer_identifier\\}\\}", savingsAccountResponse.savingsId);
```

Comment:
just to avoid confusion here, Dont use the savings id as such for the interop identifier.,   May be use a modified savings id  by adding a prefix